SpeechService/Translations.cs

```diff
@@ -689,7 +689,7 @@ public static string Humanize(decimal? value)
                     case 7:
                         return "well over " + number + " and a half " + order;
                     case 8:
-                        return "on the way to " + (number + 1) + " " + order;
+                        return "about " + (number + 1) + " " + order;
```

Comment:
Same thinking here as above. 
2801 isn't the same as 'about three thousand' but it could be 'approaching three thousand'.

Reply:
OK - I think 'Almost' is the better choice if you're dissatisfied with 'About'. 'Approaching' has the same grammar issue as 'on the way to'; "Two damage escape pods salvaged for approaching five and a half thousand credits" is incorrect. Whereas; "Two damage escape pods salvaged for almost five and a half thousand credits" and "Two damage escape pods salvaged for about five and a half thousand credits" are correct.

Reply:
What do you see as being grammatically wrong with 'approaching' (or 'on the way to', for that matter)?
I'm not sure I understand your reason for objecting to the current wording. I'll grant that 'on the way to' feels a little clunky / inefficient though.